protos/v0/jvs_plugin_service.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package abcxyz.jvs;
+
+import "jvs_request.proto";
+
+option go_package = "github.com/abcxyz/jvs/apis/v0";
+
+// JVSPlugin processes ValidateJustificationRequest and return validation
+// result.
+service JVSPlugin {
+  rpc Validate(ValidateJustificationRequest)
+      returns (ValidateJustificationResponse);
+}
+
+// ValidateJustificationRequest provides a justification for the server to validate.
+message ValidateJustificationRequest {
+
+  Justification justification = 1;
+}
+
+// ValidateJustificationResponse contains the validation result.
+message ValidateJustificationResponse {
+  bool valid = 1;
+
+  // Could be empty if it's valid.
+  // Otherwise a reason should be provided.
+  string reason = 2;
```

Comment:
It might be good to have a more structured response like:

```proto
repeated string warning = 2;
repeated string error = 3;
```

Reply:
Good idea! Updated. PTAL